ServiceBus/ServiceBus/BasketService.svc.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.ServiceModel;
+using System.Text;
+using SharedLibs.DataContracts;
+
+namespace ServiceBus
+{
+    
+    public class BasketService : IBasketService
+    {
+        /// <summary>
+        /// Method is supposed to create new basket. This is a test for database FK implementation.
+        /// </summary>
+        /// <returns>DTO object basket WITHOUT basket items.</returns>
+        public SharedLibs.DataContracts.Basket CreateBasekt()
+        {
+            try
+            {
+                using (var context = new ServiceBusDatabaseEntities())
+                {
+                    //Add new basket and two campaigns
```

Comment:
campaigns should not be mandatory. There should be BasketItems and BasketCamapigns tables with records that have FK to a basket, not the other way around (haven't check entirely, just assumption now).